aten/src/ATen/Declarations.cwrap

```diff
@@ -1748,6 +1749,7 @@
   name: _th_zero_
   cname: zero
   return: self
+  cpu_half: True
```

Comment:
Add cpu_half test maybe to test_torch?

Reply:
zero() and fill() are heavily used in tests, but its a good point about adding a specific test for half type only. this PR is landing right now but i will add these tests as a part of next [PR](https://github.com/pytorch/pytorch/pull/17376) which is in progress right now.